Makefile

```diff
@@ -88,7 +88,8 @@ logs: ## display grafana logs (follow mode)
 plugins: \
 	tree \
 	var/lib/grafana/plugins/grafana-groupedbarchart-panel-master
-plugins: ## download plugins
+plugins: ## download and install plugins
+	rsync -ar --delete src/plugins/* var/lib/grafana/plugins/
```

Comment:
We should discuss about it to have a better solution.

Reply:
Maybe mounting two distinct volumes inthe grafana container would be an easier solution? External plugins in `/var/lib/grafana/plugins/vendor` and custom plugins in `/var/lib/grafana/plugins/potsie`. WDYT?

Reply:
Thanks for your help) Grafana searches for plugins recursively - this solution works much better !